st2common/st2common/models/db/profiling.py

```diff
@@ -0,0 +1,105 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Yeah, I was thinking about that, but because some other non-model related functionality already lives in this package (db_setup, mongo access, etc.) I decided to put it there.

Ideal split would imo be, moving all the non model related db stuff to `st2common.db`. In any case, I can move it to utils for now.